tests/external_processing/test_apm.py

```diff
@@ -0,0 +1,79 @@
+from tests import test_scrubbing as base_scrubbing
```

Comment:
We'll need to revamp a little bit that, because if a change is made in tose file only, your scenario won't be tested. I'll handle that.

Reply:
Addressed in [820097b](https://github.com/DataDog/system-tests/pull/3377/commits/820097be74fdeeadcd53819addced617bbb66b4f)